variant-caller/variant-caller-wdl/calculate_contamination.wdl

```diff
@@ -17,11 +17,13 @@ workflow calulateDNAContamination {
   # Cromwell error from asking for 0 disk when the input is less than 1GB
   Int additional_disk = select_first([increase_disk_size, 20])
 
-  Float reference_size = size(ref_fasta, "GB") + size(ref_fasta_index, "GB")
-  Float cram_size = size(input_cram_file, "GB") + size(input_crai_file, "GB")
+  Float? est_ref_size
+  Float reference_size = select_first([est_ref_size, 4.0])
+  Float? est_cram_size
+  Float cram_size = select_first([est_cram_size, 30.0])
 
   String? reference_genome_version
-  String reference_genome = select_first([reference_genome_version, 'hg38'])
+  String reference_genome = select_first([reference_genome_version, "hg38"])
```

Comment:
This was changed as the WDL syntax highlighter is more compatible with double-quoted strings